src/olympia/activity/tests/test_utils.py

```diff
@@ -310,8 +309,7 @@ def _check_email(self, call, url, reason_text):
     def test_reviewer_request_for_information(self, send_mail_mock):
         AddonReviewerFlags.objects.create(
             addon=self.addon,
-            pending_info_request=datetime.now() + timedelta(
-                days=REVIEWER_NEED_INFO_DAYS_DEFAULT))
+            pending_info_request=datetime.now() + timedelta(days=14))
```

Comment:
Because it's not the same thing. I'm removing the old constant because it was meant for info requests through reviewer actions and I'm removing the ability to do that, but for now the rest of the code around info requests is kept (will be removed fully in https://github.com/mozilla/addons-server/issues/14899). Any arbitrary value would work for this test as long as it's consistent.